go/vt/sqlparser/analyzer.go

```diff
@@ -47,6 +47,15 @@ func IsValue(node ValExpr) bool {
 	return false
 }
 
+// IsNull returns true if the ValExpr is nil.
```

Comment:
I think this should say something like "IsNull returns true if the ValExpr is SQL NULL." Saying it's `nil` is misleading.